### PR TITLE
Add a few pyramid tests and some minor fixes

### PR DESCRIPTION
**Pull request by tomtor**
- Add test iterator for server tests
- test /signup and /login
- Fix Tasks worker summary spelling: with 1 coreS
- prevent creating a user with an existing name
- update development.ini

### CODE REVIEW

**Comment by ppigazzini** [comment]
The [dev-server](http://94.198.98.239/tests) and the workers are running fine since 3 hours.
I tested the registration of:
* an already registered user: OK (not permitted)
* a new user: OK (permitted)

@Stefano80 to me the merge is OK

File: fishtest/tests/test_rundb.py [hunk 1]
```diff
@@ -49,16 +53,15 @@ def test_40_list_LTC(self):
...
-      w['pending'] = False
-    rundb.buffer(run, True)
+    for run in rundb.get_runs():
+      if run['args']['username'] == 'travis' and not 'deleted' in run:
+        print('del ')
+        run['deleted'] = True
+        run['finished'] = True
+        for w in run['tasks']:
+          w['pending'] = False
+        rundb.buffer(run, True)
```
Comment (Stefano80) [inline fishtest/tests/test_rundb.py]: Should'nt we assert that we have no runs running anymore?

File: fishtest/tests/util.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from fishtest.rundb import RunDb
+
+def find_run():
```
Comment (Stefano80) [inline fishtest/tests/util.py]: Maybe we could rename this `find_travis_run` or even have a function `find_user_run(userName)` which is called with `travis` as an argument.

File: fishtest/tests/test_users.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import unittest
```
Comment (Stefano80) [inline fishtest/tests/test_users.py]: Thanks for testing, this is the spirit! Awesome!

File: fishtest/tests/util.py (hunk 2 above)
Comment (tomtor) [reply fishtest/tests/util.py]: Good suggestion, will change that

File: fishtest/tests/test_rundb.py (hunk 1 above)
Comment (tomtor) [reply fishtest/tests/test_rundb.py]: update_task will check this, so that should never be a problem

**Comment by ppigazzini** [comment]
Started the test w/ the dev-server

**Comment by tomtor** [comment]
@ppigazzini Thanks, but the only change was in the Travis tests, there should be no change in the actual running code since your test of yesterday. I will squash the commits.

**Comment by ppigazzini** [comment]
server updated, fishtest service restarted.

**Pull request merged**